backend/controllers/users.controller.js

```diff
@@ -0,0 +1,151 @@
+import User from '../models/users.model.js';
+import likedVideos from '../models/likedvideos.model.js'
```

Comment:
Can create an index.js file in the models folder.. so that there are fewer and cleaner imports..

Reply:
Done :) looks much cleaner now
Thanks